css/properties/font-size-adjust.json

```diff
@@ -15,7 +15,13 @@
               ]
             },
             "chrome_android": {
-              "version_added": false
+              "version_added": "43",
```

Comment:
font-size-adjust was enabled by default in Chrome 54. https://www.chromestatus.com/features/5730156303876096

Reply:
Separated as #4762, reverting this change!